bin/hermes-notify

```diff
@@ -116,8 +116,9 @@ def main():
     # generate and send emails
     for owner in info:
         msg = (
-            "Hello {}:\n\nYou have open Hermes labors.  Due dates, if any, "
-            "are noted with each quest.".format(owner)
+            "This email is being sent to {} because that is the owner listed\n"
```

Comment:
I think we should make this message this configurable. I don't think we should be hard-coding a reference to MDB in an open source project.